uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1012.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1012 {
+
+  /**
+   * Disables relationship types that are not necessary for CiviHR and
+   * removes existing relationships belonging to these types.
+   */
+  public function upgrade_1012() {
+    $relationshipTypesToBeDisabled = $this->up1012_getRelationshipTypesToBeDisabled();
+
+    $this->up1012_disableRelationshipTypes($relationshipTypesToBeDisabled);
+    $this->up1012_removeRelationshipsForTypes($relationshipTypesToBeDisabled);
+
+    return TRUE;
+  }
+
+  /**
+   * Returns the Ids of the the relationship types to be disabled.
+   *
+   * @return Array[int]
```

Comment:
Does the IDE not complain about this? I thought the standard way was to use `int[]`

Reply:
The IDE even makes it pretty (Atom IDE).... I'll change it anyways!